UI/org.eclipse.birt.report.designer.ui/plugin.properties

```diff
@@ -100,6 +100,7 @@ command.name.28 = reloadCssStyleCommand
 command.name.29 = CopyCellContentsAction
 command.name.30 = CreateChartViewCommand
 command.name.31 = resetImageSizeCommand
+command.name.32 = searchAction
```

Comment:
This is displayed to the user and even the other names I see above are not really appropriate as an English name to be displayed to a user.